lib/omnibus/util.rb

```diff
@@ -220,5 +290,24 @@ def create_link(a, b)
       log.debug(log_key) { "Linking `#{a}' to `#{b}'" }
       FileUtils.ln_s(a, b)
     end
+
+    private
+
+    def filter_paths
+      @filter_paths ||=
+        begin
+          # Any alternate binaries or gems from the users environment.
+          filters = Gem.paths.path.dup
+          # Any possible embedded ruby in chef products.
+          if windows?
+            filters << "C:/opscode"
+          else
+            filters << "/opt/chef" << "/opt/chefdk" << "/opt/delivery-cli"
```

Comment:
Can't you just use [`Omnibus::Project#install_dir`](https://github.com/chef/omnibus/blob/184c3f7916f0deed7f1b25ecc755c332729fc2a8/lib/omnibus/project.rb#L157-L185) as the thing you filter out? If you need to filter anything out any install_dir's for other Omnibus projects then it means the `PATH` being used to execute the Omnibus build is too promiscuous.